apps/docs/content/components/PricingOptions.mdx

```diff
@@ -0,0 +1,747 @@
+      <PricingOptions.FeatureListItem variant="excluded">
+        Access your knowledge base
+      </PricingOptions.FeatureListItem>
+      <PricingOptions.FeatureListHeading>
+        Code completion
+      </PricingOptions.FeatureListHeading>
+      <PricingOptions.FeatureListItem>
+        Code suggestions as you type
+      </PricingOptions.FeatureListItem>
+      <PricingOptions.FeatureListItem>
+        Comments to code
+      </PricingOptions.FeatureListItem>
+      <PricingOptions.FeatureListItem variant="excluded">
+        Fine-tuned models (coming soon)
+      </PricingOptions.FeatureListItem>
+    </PricingOptions.FeatureList>
+  </PricingOptions.Item>
+
+  <PricingOptions.Item>
+    <PricingOptions.Label>Available Feb 2024</PricingOptions.Label>
+    <PricingOptions.Heading>Copilot Enterprise</PricingOptions.Heading>
+    <PricingOptions.Description>
+      Copilot personalized to your organization throughout the software
+      development lifecycle. Requires GitHub Enterprise Cloud.
+    </PricingOptions.Description>
+    <PricingOptions.Price currencySymbol="$" trailingText="per user / month">
+      39
+    </PricingOptions.Price>
+    <PricingOptions.FeatureList>
+      <PricingOptions.FeatureListHeading>
+        Chat
+      </PricingOptions.FeatureListHeading>
+      <PricingOptions.FeatureListItem>
+        Unlimited messages, interactions, and history
+      </PricingOptions.FeatureListItem>
+      <PricingOptions.FeatureListItem>
+        Context-aware coding support and explanations
+      </PricingOptions.FeatureListItem>
+      <PricingOptions.FeatureListItem>
+        Debugging and security remediation assistance
+      </PricingOptions.FeatureListItem>
+      <PricingOptions.FeatureListItem>
+        Repository-based semantic search
+      </PricingOptions.FeatureListItem>
+      <PricingOptions.FeatureListItem>
+        Access your knowledge base
+      </PricingOptions.FeatureListItem>
+      <PricingOptions.FeatureListHeading>
+        Code completion
+      </PricingOptions.FeatureListHeading>
+      <PricingOptions.FeatureListItem>
+        Code suggestions as you type
+      </PricingOptions.FeatureListItem>
+      <PricingOptions.FeatureListItem>
+        Comments to code
+      </PricingOptions.FeatureListItem>
+      <PricingOptions.FeatureListItem>
+        Fine-tuned models (coming soon)
+      </PricingOptions.FeatureListItem>
+    </PricingOptions.FeatureList>
+    <PricingOptions.PrimaryAction href="/waitlist">
+      Join waitlist
+    </PricingOptions.PrimaryAction>
+  </PricingOptions.Item>
+</PricingOptions>
+```
+
+## Component props
+
+### PricingOptions <Label>Required</Label>
+
+| Name           | Type                  |  Default  | required | Description                         |
+| :------------- | :-------------------- | :-------: | :------: | :---------------------------------- |
+| `as`           | `section , div`       | `section` | `false`  | Applies the underlying HTML element |
```

Comment:
To fix 
<img width="192" alt="Screenshot 2024-03-20 at 11 48 42" src="https://github.com/primer/brand/assets/13340707/e9eed6ee-abcd-443c-9f12-dc0cce93a322">



```suggestion
| `as` | <PropTableValues values={[`section`, `div` ]} addLineBreaks /> | `section` | `false` | Applies the underlying HTML element |
```